### PR TITLE
[PlatformService] Added `AccessibilityInUseChanged`

### PR DESCRIPTION
**Pull request by mykyta-bondarenko**
It's a change for https://github.com/xamarin/md-addins/pull/5698. 
- added `AccessibilityInUseChanged ` event

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Desktop/PlatformService.cs [hunk 1]
```diff
@@ -499,7 +499,18 @@ internal virtual void RestartIde (bool reopenWorkspace)
 			proc.Start ();
 		}
 
-		public static bool AccessibilityInUse { get; protected set; }
+		static bool accessibilityInUse;
+		public static event EventHandler AccessibilityInUseChanged;
+		public static bool AccessibilityInUse {
+			get => accessibilityInUse;
+			set {
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Desktop/PlatformService.cs]: Shouldn't this remain protected set?

Comment (mrward) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Desktop/PlatformService.cs]: Good point. Did not see that 👍

Comment (mykyta-bondarenko) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Desktop/PlatformService.cs]: @mrward @Therzok thanks for pointing it out to me. I've fixed it.

**Comment by sevoku** [comment]
@monojenkins backport release-8.4

**Comment by netonjm** [comment]
Sounds good then 👍  !

**Comment by sevoku** [comment]
The `Xamarin.Azure.Tests.Functions.SdkTests.CanCancelInstallation` test failure can be ignored

**Comment by Therzok** [comment]
Rebasing the PR now will make it go green though.

**Pull request merged**